src/us/kbase/test/auth2/MongoController.java

```diff
@@ -0,0 +1,100 @@
+package us.kbase.test.auth2;
```

Comment:
What are the changes between this and the old controller class?

Reply:
The old controller class is from in kbase-common-0.0.22.jar. I cannot edit directly in this repo and was still using mongo 3.

The only change I made is delete "--nojournal" option when started mongo, which is now deprecated.

Reply:
You might want to keep that for version 3 of mongo otherwise it'll take forever to start

Reply:
I am thinking calling `MongoController` twice in `MongoStorageTestManager` where the first time get mongo version and then the second time get correct args initiated (keep --nojournal for version 3). Is there a more elegant way to do this?

Reply:
That's exactly how I would do it

Reply:
Updated!

Reply:
:+1: